themes/le-2025/layouts/partials/footer.html

```diff
@@ -42,7 +42,7 @@
       {{ if not (.Page.Params.display_inline_newsletter_embed) }}
       <div class="pt-0 md:pt-[70px]">
         <div class="border border-gray-300 rounded-lg pt-3 pb-0 px-2">
-          <h6 class="text-eyebrow mb-5">{{ i18n "newsletter_subscribe_heading" }}</h6>
+          <h4 class="text-eyebrow mb-5">{{ i18n "newsletter_subscribe_heading" }}</h4>
```

Comment:
Why this change?

Reply:
Heading elements are now in a sequentially-descending order.